src/App.m.css

```diff
@@ -6,5 +6,5 @@
 .content {
 	flex-grow: 1;
 	display: flex;
-	margin-top: 80px;
+	margin-top: 78px;
```

Comment:
Why 78 vs 80? If we use 80 we can use `--grid-size`

Reply:
I'm going to use a unified `--header-size` var :+1: